prefect_gcp/secret_manager.py

```diff
@@ -270,3 +284,91 @@ def example_cloud_storage_delete_secret_version_flow():
     partial_destroy = partial(client.destroy_secret_version, name=name, timeout=timeout)
     await to_thread.run_sync(partial_destroy)
     return name
+
+
+class SecretManager(SecretBlock):
```

Comment:
This block represents a single secret stored in GCP Secrets Manager, so I think we should name it just `GcpSecret`.
```suggestion
class GcpSecret(SecretBlock):
```